pkg/process/util/address.go

```diff
@@ -67,90 +85,38 @@ func ToLowHigh(addr Address) (l, h uint64) {
 	return
 }
 
-type v4Address [4]byte
-
 // V4Address creates an Address using the uint32 representation of an v4 IP
 func V4Address(ip uint32) Address {
-	var a v4Address
-	a[0] = byte(ip)
-	a[1] = byte(ip >> 8)
-	a[2] = byte(ip >> 16)
-	a[3] = byte(ip >> 24)
-	return a
+	return Address{
+		netaddr.IPv4(
+			uint8(ip),
+			uint8(ip>>8),
+			uint8(ip>>16),
+			uint8(ip>>24),
+		),
+	}
 }
 
 // V4AddressFromBytes creates an Address using the byte representation of an v4 IP
 func V4AddressFromBytes(buf []byte) Address {
-	var a v4Address
+	var a [4]byte
 	copy(a[:], buf)
-	return a
-}
-
-// Bytes returns a byte array of the underlying array
-func (a v4Address) Bytes() []byte {
-	return a[:]
+	return Address{netaddr.IPFrom4(a)}
```

Comment:
I found out that starting in Go 1.17, we can do this without the copy:
`return Address{netaddr.IPFrom4(*(*[4]byte)(buf))}`